src/data/contributors/Kalyani.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Kalyani Kolhe",
+    "skills" : ["C", "C++", "HTML", "JS"],
+    "cause" : "I code to develop my skills",
+    "image" : "Avatar4",
+    "github" : "https://github.com/kalyani489",
+    "linkedin" : "https://www.linkedin.com/in/kalyani-kolhe-05184a160"
+}
```

Comment:
enter a new line here :)
A new line should ideally be added to the end of every file.

Reply:
Yes sure, I got it.
I'll resolve it and again pull a request

Reply:
@Kalyani489, you can just make changes to this branch only and commit and push it. It will automatically get to us within this PR.